src/lib/app-switcher/app-switcher.component.html

```diff
@@ -0,0 +1,24 @@
+<button mat-icon-button [matMenuTriggerFor]="applicationMenu">
+  <mat-icon>apps</mat-icon>
+</button>
+<mat-menu #applicationMenu="matMenu" [overlapTrigger]="false" class="appswitcher-menu">
+  <div class="brand-container">
+    <mat-icon class="logo-icon" [svgIcon]="brandIcon"></mat-icon>
+    <div class="mat-body">{{ brandText }}</div>
+  </div>
+
+  <ng-container *ngFor="let product of productList; last as isLast">
+    <mat-divider *ngIf="product.divider"></mat-divider>
+    <a mat-menu-item *ngIf="product.icon; else regularItem" [href]="product.href">
+      <mat-icon>{{ product.icon }}</mat-icon>
+      {{ product.text }}
+    </a>
+
+    <ng-template #regularItem>
+      <a mat-menu-item class="explore-more mat-caption">
```

Comment:
I would recommend this also not to be configurable. Its just the footer of the app.

Reply:
We could separate this but still need to pass link for explore more since this could potentially change. Can call it footerLink? also same comment above with translations

Reply:
Yeah, i thought the URL was gonna be something from `teradata.com` that wouldnt change, but if thats the case, then making it configurable would be good.